tracer/src/Datadog.Trace/Telemetry/Metrics/MetricTags.cs

```diff
@@ -272,6 +273,7 @@ public enum IastInstrumentedSinks
         [Description("vulnerability_type:weak_randomness")] WeakRandomness = 12,
         [Description("vulnerability_type:hardcoded_secret")] HardcodedSecret = 13,
         [Description("vulnerability_type:xcontenttype_header_missing")] XContentTypeHeaderMissing = 14,
+        [Description("vulnerability_type:trust_boundary_violation")] TrustBoundaryViolation = 15,
```

Comment:
again, just curious, but why do we use underscores here and not above for the integration name?

Reply:
That's a question for @NachoEchevarria